app/services/state_file/json_return_sample_service.rb

```diff
@@ -0,0 +1,61 @@
+module StateFile
+  class JsonReturnSampleService
+    def initialize
+      @samples = {}
+      @old_sample = "app/controllers/state_file/questions/df_return_sample.json"
+    end
+
+    def samples
+      load_samples
+      @samples
+    end
+
+    def self.key(us_state, sample_name)
+      "#{us_state}_#{sample_name}"
+    end
+
+    def self.label(sample_name)
+      sample_name.humanize
+    end
+
+    def include?(key)
+      path(key).present?
+    end
+
+    def read(key)
+      File.read(path(key)) if include?(key)
+    end
+
+    def old_sample
```

Comment:
can we also put in a "TODO" here to re-evaluate and remove use of `old_sample` when we are only using updated return examples?